src/xy/xyCovariance.ts

```diff
@@ -0,0 +1,37 @@
+import mean from 'ml-array-mean';
```

Comment:
Should be an internal dependence and not call another package. You may migrate this method as well if we don't have it yet.

You may actually add all the methods from https://github.com/mljs/array/tree/master/packages in the folder 'x' (and rename the functions so that they start with 'x')